design/comments.go

```diff
@@ -36,6 +36,9 @@ var commentAttributes = a.Type("CommentAttributes", func() {
 	a.Attribute("body", d.String, "The comment body", func() {
 		a.Example("This is really interesting")
 	})
+	a.Attribute("body.rendered", d.String, "The comment body rendered in HTML", func() {
```

Comment:
We could add comment or create a type for the input comment (vs output with the new `body.rendered` element). What do you think ?